### PR TITLE
[tests-only][full-ci]Remove body 'doesnotmatter' from the request not needing it

### PR DESCRIPTION
**Pull request by SagarGi**
## Description
This PR removes body 'doesnotmatter' in the request which does not require it. 

## Related Issue

- https://github.com/owncloud/core/issues/40126

## How Has This Been Tested?

- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/AuthContext.php [hunk 1]
```diff
@@ -189,14 +189,14 @@ public function verifyStatusCode(string $ocsCode, string $httpCode, string $endP
...
-	 * @param string $body
+	 * @param ?string $body
 	 * @param string $ofUser
 	 * @param TableNode $table
 	 *
 	 * @return void
 	 * @throws Exception
 	 */
-	public function userRequestsEndpointsWithBodyAndNoAuthThenStatusCodeAboutUser(string $method, string $body, string $ofUser, TableNode $table):void {
+	public function userRequestsEndpointsWithBodyAndNoAuthThenStatusCodeAboutUser(string $method, ?string $body, ?string $ofUser, TableNode $table):void {
```
Comment (kiranparajuli589) [inline tests/acceptance/features/bootstrap/AuthContext.php:199]: is `$ofUser` now optional? imo no change is required here.

**Review by phil-davis — APPROVED** [review]
LGTM

File: tests/acceptance/features/bootstrap/AuthContext.php (hunk 1 above)
Comment (phil-davis) [reply tests/acceptance/features/bootstrap/AuthContext.php:199]: This now gets called sometimes with body `null`. And `sendRequest` can already be passed body `null`.

**Comment by phil-davis** [comment]
@SagarGi @kiranparajuli589 please get someone to bump the core commit id in reva (edge and master) and oCIS. It will be good to have these test changes running in that CI.

**Pull request merged**